lib/data_directory.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+
+const normalizeFilePath = filePath => {
+  if (typeof filePath !== 'string') {
+    throw new TypeError(`Expected a string, got ${typeof filePath}`);
+  }
+
+  const homeDirectory = os.homedir();
+
+  if (homeDirectory) {
+    return filePath.replace(/^~(?=$|\/|\\)/, homeDirectory);
```

Comment:
Is there a path helper that sanitize strings instead of a regex? I always prefer platform agnostic helper/libs for those sort of things